### ISSUE TITLE
Location auf aktuelles Schema anpassen

### ISSUE THREAD

**Issue by JirkaDellOro** [Rina14/Abschlussarbeit#4]
Location war der erste Bestandteil, den ich implementiert hatte, da noch mit klassischer Konstruktorsyntax
```typescript 
export let locations = {
    window: new Theater.Location("Window", "../Backgrounds/bg_window.png"),
    rain: new Theater.Location("Whatever", "../Backgrounds/bg_city_rain.png")
  };
```

Dann schien aber die Objektsyntax intuitiver, wie ich sie bei den Character und Menü-Optionen verwende. Daher würde ich Location auch darauf umstellen wollen, damit es etwa so aussieht:

```typescript
export let locations = {
    window: {
       name: "Window", 
       background: "../Backgrounds/bg_window.png",
       foreground: "../xxxx",
    }
    cityRain: {
       name: "Rain in the City", 
       background: "../Backgrounds/bg_city_rain.png",
       foreground: "../xxxx",
    }
  };
```

Gibt es eine Meinung dazu? Ich muss auch noch prüfen, ob es noch Restriktionen gibt, die dagegen sprechen.

**Comment by Rina14** [comment]
Hmm, aber wieso? Das erste scheint mir sinnvoller, da es ja nur einen background gibt und dazu keinen foreground. Davor sind nur Charaktere und ggf. Objekte/Gegenstände. Würde ich mit foreground dann Objekte einfügen und anzeigen können oder wie?

**Comment by JirkaDellOro** [comment]
Meine Frage bezieht sich nicht auf Fore-/Background, das gibt es auch jetzt schon. Hast Du nur bislang nicht genutzt und musst Du auch nicht mit der Objektsyntax machen. Es geht allein um die Schreibweise, also mit 
`new Theater.Location(....)` oder mit 
`{name: ..., background: ..., ...}`

Ich meine, es wäre besser ein Schema für alles zu haben statt für die Locations etwas Spezielles.

**Comment by Rina14** [comment]
Stimmt, hab's grad ausprobiert, my bad. Und ja mir ging es tatsächlich zuerst auch um die Syntax. Da fand ich Ersteres einfacher, aber ehrlich gesagt ist das Zweite schematisch doch besser, auch weil man dann direkt auswählen kann, was man genau will (rainy, cloudy usw.).

Also falls möglich, fände ich es gut.
